docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
         volumes:
             - ./:/var/www/symfony:cached
             - ./var/logs/symfony:/var/www/symfony/var/logs:cached
+            - ./php-fpm/php-limit.ini:/usr/local/etc/php/conf.d/php-limit.ini
```

Comment:
Docker configuration moved to the /docker folder since #38 
Please move this files inside the docker/php-fpm folder :)

Reply:
(first rebase to see new folder, then change path)